Slambook/index.js

```diff
@@ -1,14 +1,26 @@
 let form;
 
 let allSlamArray=[];
+let formValue = {};
+let err ;
 //Write code below to select the form element
 form = document.querySelector('form')
 console.log(form,"form");
 // Write your code above
 
```

Comment:
Instead of directly setting the style property of the 'tag' element, consider using specific style properties like 'tag.style.display = 'block'' for better clarity and consistency with other style modifications.